genai-perf/genai_perf/config/generate/perf_analyzer_config.py

```diff
@@ -338,7 +342,8 @@ def _add_inference_load_args(self, config: ConfigCommand) -> List[str]:
     def _add_prompt_source_args(self, config: ConfigCommand) -> List[str]:
         prompt_source_args = []
         if (
-            config.input.prompt_source == PromptSource.PAYLOAD
+            hasattr(config.input, "prompt_source")
```

Comment:
This isn't necessary. The only reason this is missing is because it wasn't created during the inference step. See my comment about the failure to infer options in the config file (in parser.py).

This applies to all three changes made to this file.